tests/test_fixtures/rule_based_profiler/alice_user_workflow_verbose_profiler_config.yml

```diff
@@ -126,7 +126,7 @@ rules:
         module_name: great_expectations.rule_based_profiler.expectation_configuration_builder
         column: $domain.domain_kwargs.column
         min_value: $variables.min_timestamp
-        max_value: $parameter.my_max_event_ts.value[0]
+        max_value: $parameter.my_max_event_ts.value[-1]
```

Comment:
❤️ I will keep this pattern in mind...

Reply:
@cdkini Yes -- the reason this is important, is because we want the "most recent batch".  Thanks!